BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -69,40 +77,22 @@ enum FeatureFlag: String, Codable {
              .enableCipherKeyEncryption,
              .nativeCarouselFlow,
              .nativeCreateAccountFlow,
-             .testLocalFeatureFlag:
+             .testLocalFeatureFlag,
+             .testLocalInitialBoolFlag,
+             .testLocalInitialIntFlag,
+             .testLocalInitialStringFlag:
             false
         case .emailVerification,
-             .testInitialBoolFlag,
-             .testInitialIntFlag,
-             .testInitialStringFlag,
-             .testRemoteFeatureFlag:
+             .testRemoteFeatureFlag,
+             .testRemoteInitialBoolFlag,
+             .testRemoteInitialIntFlag,
+             .testRemoteInitialStringFlag:
             true
         }
     }
 
     /// The display name of the feature flag.
     var name: String {
-        switch self {
-        case .emailVerification:
-            "Email Verification"
-        case .enableAuthenticatorSync:
-            "Enable Authenticator Sync"
-        case .enableCipherKeyEncryption:
-            "Enable Cipher Key Encryption"
-        case .nativeCarouselFlow:
-            "Native Carousel Flow"
-        case .nativeCreateAccountFlow:
-            "Native Create Account Flow"
-        case .testLocalFeatureFlag:
-            "Test Local Feature Flag"
-        case .testRemoteFeatureFlag:
-            "Test Remote Feature Flag"
-        case .testInitialBoolFlag:
-            "Test Initial Boolean Flag"
-        case .testInitialIntFlag:
-            "Test Initial Integer Flag"
-        case .testInitialStringFlag:
-            "Test Initial String Flag"
-        }
+        rawValue.split(separator: "-").map(\.localizedCapitalized).joined(separator: " ")
```

Comment:
🤔 I think this will show `EnableCipherKeyEncryption` instead of `Enable Cipher Key Encryption` but I guess it's fine for an internal debug menu not to add much more complexity to this. But if we'd like to add the case, I think for now it's just easier to add a specific if for that returning the value hardcoded than adding the logic to detect the uppercase letter to add the spacing.
```suggestion
        guard .enableCipherKeyEncryption != self else {
            return "Enable Cipher Key Encryption"
        }
        return rawValue.split(separator: "-").map(\.localizedCapitalized).joined(separator: " ")
```

Reply:
Yeah, I thought about including something fancier, but since it's an internal debug menu *and* that particular flag is hidden from the menu already, I figured it wasn't worth the additional code (and attendant complexity). If we want to put it on the menu, we can add the hardcoded name when we do that.